Example/Custom Integration (ObjC)/README.md

```diff
@@ -1,7 +1,18 @@
 # Custom Integration (ObjC)
 
-This example app demonstrates how to to use `STPAPIClient` to accept various payment methods. These integrations may be a useful reference if you're building your own payment UI, and not using `STPPaymentContext`. 
+This example app demonstrates how to to use `STPAPIClient` to accept various payment methods. This may be a useful reference if you're building your own payment UI and not using `STPPaymentContext`.
```

Comment:
why "These integrations" to "This"?

Reply:
I think I originally changed it because this doc refers to one of the 3 example apps and doesn't talk about the existence of the other two :). What do you think?